spec/lib/api/2.0/skus-spec.js

```diff
@@ -0,0 +1,360 @@
+            expect(sku).to.have.property('discoveryGraphName')
+            .that.equals(input.discoveryGraphName);
+        });
+
+        it('should have the correct discoveryGraphOptions', function () {
+            expect(sku).to.have.property('discoveryGraphOptions')
+            .that.deep.equals(input.discoveryGraphOptions);
+        });
+
+        it('should have regenerated SKUs', function () {
+            expect(workflowApiService.createAndRunGraph).to.have.been.calledOnce;
+            expect(workflowApiService.createAndRunGraph)
+                .to.have.been.calledWith({
+                    name: 'Graph.GenerateSku',
+                    options: {
+                        defaults: {
+                            nodeId: node.id
+                        }
+                    }
+                });
+        });
+
+        it('should contain the new sku in GET /skus', function () {
+            return helper.request().get('/api/2.0/skus')
+            .expect('Content-Type', /^application\/json/)
+            .expect(200, [sku]);
+        });
+
+        it('should return the same sku from GET /skus/:id', function () {
+            return helper.request().get('/api/2.0/skus/' + sku.id)
+            .expect('Content-Type', /^application\/json/)
+            .expect(200, _.merge({}, sku, { packInfo: null }) );
+        });
+
+        describe('PATCH /skus/:id', function () {
+            var updated;
+
+            beforeEach('reset createAndRunGraph stub', function () {
+                workflowApiService.createAndRunGraph.reset();
+            });
+
+            beforeEach('PATCH /skus/:id', function () {
+                sku.name = 'updated sku name';
+                return helper.request().patch('/api/2.0/skus/' + sku.id)
+                .send(sku)
+                .expect('Content-Type', /^application\/json/)
+                .expect(200)
+                .then(function (res) {
+                    updated = res.body;
+                });
+            });
+
+            it('should have an updated name', function () {
+                expect(updated).to.have.property('name').that.equals(sku.name);
+            });
+
+            it('should have the same rules', function () {
+                expect(updated).to.have.property('rules').that.deep.equals(sku.rules);
+            });
+
+            it('should have regenerated SKUs', function () {
+                expect(workflowApiService.createAndRunGraph).to.have.been.calledOnce;
+                expect(workflowApiService.createAndRunGraph)
+                    .to.have.been.calledWith({
+                        name: 'Graph.GenerateSku',
+                        options: {
+                            defaults: {
+                                nodeId: node.id
+                            }
+                        }
+                    });
+            });
+        });
+
+        describe('GET /skus/:id/nodes', function () {
```

Comment:
Hi Felix,

I used the same structure as of  the test file of the of the 1.1 API (spec/lib/api/1.1/skus-spec.js ), I didnt want to change it in order to consistency